plugin-cpuload/translations/cpuload_de.desktop

```diff
@@ -1,13 +1,2 @@
-[Desktop Entry]
-Type=Service
-ServiceTypes=LxQtPanel/Plugin
-
-Name=Cpu monitor
-Comment=Displays the current CPU load.
-
-
-
-
-# Translations
-Comment[de]=Prozesslast-Monitor
 Name[de]=Prozessauslastung
```

Comment:
Is it an intention to have `Processauslastung` vs. `Processorauslastung` on different places?